package.json

```diff
@@ -76,8 +76,10 @@
     "@opentelemetry/core": "<1.4.0",
     "crypto-randomuuid": "^1.0.0",
     "diagnostics_channel": "^1.1.0",
+    "fnv-plus": "^1.3.1",
```

Comment:
Do we need this? If we can use any hashing algorithm, why not use the same as what AppSec is using?

Reply:
which one is AppSec using?

Reply:
Specifically, fnv-plus implements the FNV64 1a hash while the "spec" basically says we should use FNV64 1 otherwise we must use some sort of consistent hash. Since FNV64 1a is no better than any other hash, say CRC32 or SHA1, then we might as well just use an existing hash and reduce our dependency count.

So, we'll figure out what hash appsec already introduced and switch to that.

Reply:
Hashing algorithm appsec is using is [here](https://github.com/DataDog/dd-trace-js/blob/e4b9a268f0429b6f1e92c384b61a0d104aeb1259/packages/dd-trace/src/appsec/iast/analyzers/vulnerability-analyzer.js#L113), I'm not sure if it's a good candidate, or maybe we should just use a built-in algo

Reply:
We have decided to use a built-in algo, will update it ^ ^

Reply:
We can keep the cryptographic hash for now since there is a cache around it. It's not a big issue if it's slow.